kafka/config.go

```diff
@@ -91,13 +93,29 @@ func value2string(v ConfigValue) (ret string, errstr string) {
 		ret = fmt.Sprintf("%d", x)
 	case string:
 		ret = x
+	case types.Slice:
+		ret = ""
+		arr := v.([]ConfigValue)
+		for _, i := range arr {
+			temp, err := value2string(i)
+			if err != "" {
+				ret = ""
+				errstr = fmt.Sprintf("Invalid value type %T", v)
+				break
```

Comment:
nit:why not just return here? 
return nil, fmt.Sprintf("Invalid value type %T", v)

and avoid the whole variable errstr as it was earlier

Reply:
I thought reducing the number of returns would make it more readable and predictable.